docs/container-register-disks.md

```diff
@@ -45,7 +45,7 @@ else a user might want to distribute with their VMIs.
 
 ### High Level Design
 
-**Standardised KubeVirt VMI disk Wrapper**
+**Standardized KubeVirt VMI disk Wrapper**
```

Comment:
Isn't this UK vs US English?

Reply:
@xpivarc thanks for approving & merging -> :smile:  -> it may be :thinking: a UK vs US English element -> I had just seen it, writen as "standardized", later within that markdown file and thought to maybe just make it the same in the bold-text, within the High Level Design heading.